flent/runners.py

```diff
@@ -680,8 +680,14 @@ def find_binary(self, **args):
             args['socket_timeout'] = ""
 
         if args['test'] in ("TCP_STREAM", "TCP_MAERTS", "omni"):
+            ss_args = {}
+            for k, v in args:
+                if k in ["host", "interval", "length", "target", "ip_version"]:
+                    ss_args[k] = v
+            self.add_child(SsRunner, ss_args)
```

Comment:
You'll need to also pass the two other positional arguments to ProcessRunner (which is where the init method that will be called for SsRunner lives). Delay is in `self.delay` and `remote_host` should be None (because that is passed as the 'host' arg above).